packages/dart/noports_core/pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
   at_commons: ^5.6.1
   at_cli_commons: ^3.0.1
   at_utils: ^3.0.19
+  at_base2e15: ^1.0.0
+  shrink: ^1.0.0
```

Comment:
we use shrink to shrink the event payloads, and we encode the resulting bytes using base2e15